reservation_system/guests.py

```diff
@@ -72,11 +72,15 @@ def create():
             flash(format_required_field_error(error_fields))
         else:
             db = get_db()
-            db.execute(
+            cursor = db.execute(
                 f"INSERT INTO {table} ({columns}) VALUES ({placeholders})",
                 data,
             )
+            guest_id = cursor.lastrowid
             db.commit()
+
+            if location := request.args.get("redirect"):
```

Comment:
Is there a cleaner way to do this?
Also, "Go back" button on forms returns to parent page and not the originating page.
For example, calendar -> edit guest -> go back -> take you to guest view and not back to the calendar.